MediaPlayer/MediaPlayer.js

```diff
@@ -328,6 +336,7 @@ const MediaPlayerBase = kind({
 					paused={paused}
 					repeat={repeat}
 					shuffle={shuffle}
+					skin={skin}
```

Comment:
this is not needed anymore once the suggestion from MediaControls is implemented

Reply:
Solved